Gems/Atom/RPI/Code/Source/RPI.Builders/Model/MorphTargetExporter.cpp

```diff
@@ -265,51 +265,71 @@ namespace AZ::RPI
         metaData.m_numVertices = numMorphedVertices;
 
         const float morphedVerticesRatio = numMorphedVertices / static_cast<float>(numVertices);
-        AZ_Printf(ModelAssetBuilderComponent::s_builderName, "'%s' morphs %.1f%% of the vertices.", blendShapeName.c_str(), morphedVerticesRatio * 100.0f);
+        
 
-        // Calculate the minimum and maximum position compression values.
+        if (numMorphedVertices > 0)
         {
-            const AZ::Vector3& boxMin = deltaPositionAabb.GetMin();
-            const AZ::Vector3& boxMax = deltaPositionAabb.GetMax();
-            auto [minValue, maxValue] = AZStd::minmax({boxMin.GetX(), boxMin.GetY(), boxMin.GetZ(),
-                boxMax.GetX(), boxMax.GetY(), boxMax.GetZ()});
-
-            // Make sure the diff between min and max isn't too small.
-            if (maxValue - minValue < 1.0f)
+            AZ_Printf(
+                ModelAssetBuilderComponent::s_builderName,
+                "Blend shape named '%s' morphs %.1f%%  (%" PRIu32 " / % " PRIu32 ") of the vertices of mesh %s ",
+                blendShapeName.c_str(),
+                morphedVerticesRatio * 100.0f,
+                numMorphedVertices,
```

Comment:
Authors note:  This entire block is part of the code that discards blend shapes (morphs) that target 0 vertices in the model.  This creation of of 0-vert-blendshapes is what would later cause an assert if the model was loaded.